### PR TITLE
VDPN-2  Create login page with functionality

### PR DESCRIPTION
**Pull request by dragusandreea**

### CODE REVIEW

File: TravelPoints-UI/src/app/app-routing.module.ts
Comment (iDietzer) [inline TravelPoints-UI/src/app/app-routing.module.ts:1]: M: why are you removing this files if the file created below has the same content?

File: TravelPoints-UI/src/app/components/login-dialog/login-dialog.component.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        this.dialogRef.close();
+        alert("Login successful!")
+        this.router.navigate(['/destinations'])
+      },
+      error: () => alert("Login failed")
+    })
+  }
+
+  openRegisterDialog() {
+    this.dialog.open(LoginDialogComponent, {
```
Comment (iDietzer) [inline TravelPoints-UI/src/app/components/login-dialog/login-dialog.component.ts]: S: when adding the registration form this should be changed

File: TravelPoints-UI/src/app/components/login-dialog/login-dialog.component.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      width: '1000px'
```
Comment (iDietzer) [inline TravelPoints-UI/src/app/components/login-dialog/login-dialog.component.ts]: W: as a best practice width and height should never be in pixels; for width use vw and and for height use vh https://dev.to/segunolatunji/mastering-css-sizing-a-guide-to-responsive-design-49hd

**Review by iDietzer — APPROVED** [review]

File: TravelPoints-UI/src/app/components/login-dialog/login-dialog.component.ts (hunk 1 above)
Comment (dragusandreea) [reply TravelPoints-UI/src/app/components/login-dialog/login-dialog.component.ts]: obviously :)

**Pull request merged**